squiggy/models/course.py

```diff
@@ -114,8 +114,8 @@ def get_advanced_asset_search_options(cls, course_id):
             'categories': [],
         }
         sql = """
-            SELECT id, CONCAT(category_name, ' - ', name)
-            AS label FROM course_groups
+            SELECT id, CONCAT(category_name, ' - ', name) AS label 
+            FROM course_groups
             WHERE course_id = :course_id
             ORDER BY category_name, name
```

Comment:
@mlincoln - I think you have trailing whitespace here. The linter (python) is complaining.